app/models/comfy/cms/page.rb

```diff
@@ -3,14 +3,28 @@
 class Comfy::Cms::Page < ActiveRecord::Base
   scope :most_popular, -> (number_of_items) { unscoped.order('page_views desc').limit(number_of_items) }
 
-  def self.all_english_articles
+  scope :with_tags, -> (tags) { joins(:taggings).where('tag_id IN (?)', tags.map(&:id)) }
+
+  scope :remove_self_from_results, -> (page) { where.not(id: page.id) }
+
+  scope :remove_alternate_language_page_when_welsh, (lambda do |page|
```

Comment:
You probably already went over this in person, but this was the effect of the 'max line length' rubocop rule, and the 'Use the `lambda` method for multi-line lambdas.' rule. So if a lambda was over a certain length, it needed to be broken onto a couple of lines, and then needed to use the keyword syntax.